libs/platform/src/lib/components/form/combobox/pipes/highlight.pipe.ts

```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ComboboxConfig, MatchingStrategy } from '../combobox.config';
+
+@Pipe({
+    name: 'highlight'
+})
+export class HighlightPipe implements PipeTransform {
```

Comment:
There is another "highlighting" pipe transform in core called `SearchHighlightPipe`:

https://github.com/SAP/fundamental-ngx/blob/master/libs/core/src/lib/utils/pipes/search-highlight.pipe.ts

This is a more general implementation, because it allows for different matching strategies. Since highlighting is a common feature for both libraries, is it possible to reduce code duplication?

Can we create a shared module with the `searchByStrategyStartsWith` and `searchByStrategyContains` and have both pipes import these functions?